numpy/lib/utils.py

```diff
@@ -151,9 +151,6 @@ def deprecate(*args, **kwargs):
 
     >>> olduint = np.deprecate(np.uint)
     >>> olduint(6)
-    /usr/lib/python2.5/site-packages/numpy/lib/utils.py:114:
-    DeprecationWarning: uint32 is deprecated
-      warnings.warn(str1, DeprecationWarning, stacklevel=2)
```

Comment:
This doesn't look right - the point of this example is to show the deprecation warning

Reply:
I restored the basic deprecation warning, but added a variable flag because:
1) Refguide seems to capture / suppress the warning internally
2) We're deprecating an unsized type so portability isn't great anway